staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go

```diff
@@ -29,13 +29,13 @@ import (
 // indicating that the controller identified by controllerName is waiting for syncs, followed by
 // either a successful or failed sync.
 func WaitForCacheSync(controllerName string, stopCh <-chan struct{}, cacheSyncs ...cache.InformerSynced) bool {
-	klog.Infof("Waiting for caches to sync for %s controller", controllerName)
+	klog.InfoS("Waiting for caches to sync", "controller", controllerName)
```

Comment:
```suggestion
	klog.InfoS("Waiting for caches to sync", "controlleNamer", controllerName)
```

Ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments